src/app/app.component.html

```diff
@@ -1 +1,6 @@
-<app-map></app-map>
\ No newline at end of file
+<app-heading></app-heading>
```

Comment:
Do Angular components not need to be wrapped in one div like it does in React? For example I would've thought it needed to look liked this

```
<div>
  <app-heading></app-heading>
  <div class="map-container">
    <app-map></app-map>
  </div>
</div>
```
Or am I missing something?

Reply:
I do not believe it does since it is compiled into a body under the hood.